UnitySDK/Assets/ML-Agents/Plugins/Barracuda.Core/Tools.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
I think we should remove the Tools folder now that we included the python converter into the python side of mlagents

Reply:
The Tools folder will be left here for iteration and will be removed before the release. Leaving this comment unresolved so we do not forget

Reply:
Waiting for Rej to integrate your changes into Barracuda distribution then will push new version of Barracuda and remove this folder

Reply:
Tools are gone by now